DQMOffline/RecoB/interface/JetTagPlotter.h

```diff
@@ -80,6 +84,12 @@ class JetTagPlotter : public BaseBTagPlotter {
 
   // reconstructed jet phi
   FlavourHistograms<double> * dJetRecPhi;
+  
+  // jet Phi larger than requested discrimnator cut
+  FlavourHistograms<double> * dJetPhiDiscrCut;
```

Comment:
indeed, any reason its not trivial for these three variables and can be done now while you are thinking about it?